testhelpers/assertions.go

```diff
@@ -175,3 +187,18 @@ func (a AssertionManager) Error(actual error) {
 		a.testObject.Fatal("Expected an error but got nil")
 	}
 }
+
+func (a AssertionManager) ErrorContains(actual error, expected string) {
+	a.testObject.Helper()
+
+	if actual == nil {
+		a.testObject.Fatalf("Expected %q an error but got nil", expected)
+	}
+
+	a.Contains(actual.Error(), expected)
+}
+
+func cutSuffixWhitespace(input string) string {
+	re := regexp.MustCompile(`  +$`)
```

Comment:
I think it would be clearer to use the whitespace character `\s` here. I'm also not sure if this would match a situation where there was only one space character at the end of the line, as it appears there are two spaces in this regex. Perhaps I'm missing a case where that's useful?

Reply:
@simonjjones brings up some good points about this being a little confusing. 

From a performance perspective, I would think that compiling a regex everytime this is called isn't optimal either.

Is there a reason something like [`strings.TrimRight(input, "\t \n")`](https://golang.org/pkg/strings/#TrimRight) wouldn't work here? From what I can tell, inputs to this method are lines (not multiline).


If we wanted to keep it exactly inline with the previous implementation the following should be equivalent:

```
strings.TrimRightFunc(input, func(c rune) bool { return unicode.IsSpace(c); })
```

Additional references: 

* https://www.admfactory.com/how-to-trim-whitespace-from-string-in-golang/